lib/test/t_FaureSequence_std.cxx

```diff
@@ -31,31 +31,74 @@ int main(int, char *[])
 
   try
   {
-    // Create a Faure sequence
+    // Create 8 points from a Faure sequence in dimension 1
+    Sample expected1D(8, 1);
+    expected1D[0] = Point({1.0 / 2.0});
+    expected1D[1] = Point({1.0 / 4.0});
+    expected1D[2] = Point({3.0 / 4.0});
+    expected1D[3] = Point({1.0 / 8.0});
+    expected1D[4] = Point({5.0 / 8.0});
+    expected1D[5] = Point({3.0 / 8.0});
+    expected1D[6] = Point({7.0 / 8.0});
+    expected1D[7] = Point({1.0 / 16.0});
+    FaureSequence sequence1D(1);
+    fullprint << sequence1D << std::endl;
+    Sample faureSample1D(sequence1D.generate(8));
+    assert_almost_equal(faureSample1D, expected1D);
+
+    // Create 8 points from a Faure sequence in dimension 2
+    Sample expected2D(8, 2);
+    expected2D[0] = Point({0.5, 0.5});
+    expected2D[1] = Point({0.25, 0.75});
+    expected2D[2] = Point({0.75, 0.25});
+    expected2D[3] = Point({0.125, 0.625});
+    expected2D[4] = Point({0.625, 0.125});
+    expected2D[5] = Point({0.375, 0.375});
+    expected2D[6] = Point({0.875, 0.875});
+    expected2D[7] = Point({0.0625, 0.9375});
+    FaureSequence sequence2D(2);
+    fullprint << sequence2D << std::endl;
+    Sample faureSample2D(sequence2D.generate(8));
+    assert_almost_equal(faureSample2D, expected2D);
+
+    // Create 8 points from a Faure sequence in dimension 3
+    Sample expected3D(8, 3);
+    expected3D[0] = Point({1.0 / 3.0, 1.0 / 3.0, 1.0 / 3.0});
+    expected3D[1] = Point({2.0 / 3.0, 2.0 / 3.0, 2.0 / 3.0});
+    expected3D[2] = Point({1.0 / 9.0, 4.0 / 9.0, 7.0 / 9.0});
+    expected3D[3] = Point({4.0 / 9.0, 7.0 / 9.0, 1.0 / 9.0});
+    expected3D[4] = Point({7.0 / 9.0, 1.0 / 9.0, 4.0 / 9.0});
+    expected3D[5] = Point({2.0 / 9.0, 8.0 / 9.0, 5.0 / 9.0});
+    expected3D[6] = Point({5.0 / 9.0, 2.0 / 9.0, 8.0 / 9.0});
+    expected3D[7] = Point({8.0 / 9.0, 5.0 / 9.0, 2.0 / 9.0});
+    FaureSequence sequence3D(3);
```

Comment:
Have you checked the values for larger samples, even if these values don't have to be checked in the test? This is just the beginning of the sequence and surprises can pop up for larger values...